examples/tools/system/package_manager.rst

```diff
@@ -0,0 +1,200 @@
+.. _examples_tools_system_package_manager:
+
+Wrapping system requirements in a Conan package
+===============================================
+
+Conan can manage system packages, allowing you to install platform-specific dependencies easily.
+This is useful when you need to install platform-specific system packages.
+For example, you may need to install a package that provides a specific driver or graphics library that only works on a specific platform.
+
+Conan provides a way to install system packages using the :ref:`system package manager<conan_tools_system_package_manager>` tool.
+
+In this example, we are going to explore the steps needed to package a system library and what is needed to consume it in a Conan package.
+For this illustration, we are going to package a simple C++ application that uses the `ncurses <https://invisible-island.net/ncurses/>`_ library.
+
+Please, first clone the sources to recreate this project. You can find them in the
+`examples2 repository <https://github.com/conan-io/examples2>`_ on GitHub:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/examples/tools/system/package_manager/
+
+
+You may find the following tree structure:
+
+.. code-block:: text
+
+    .
+    ├── conanfile.py
+    └── consumer
+        ├── CMakeLists.txt
+        ├── conanfile.py
+        └── ncurses_version.cpp
+
+
+The **conanfile.py** file is the recipe that packages the ncurses library.
+Finally, the **consumer** directory contains a simple C++ application that uses the ncurses library, we will visit it later.
+
+When packaging a pre-built system library, we do not need to build the project from source, but only to install the
+system library and package its information.
+In this case, we are going to check the **conanfile.py** file that packages the ncurses library first:
+
+.. code-block:: python
+
+    from conan import ConanFile
+    from conan.tools.system import package_manager
+    from conan.tools.gnu import PkgConfig
+    from conan.errors import ConanInvalidConfiguration
+
+    required_conan_version = ">=2.0"
+
+
+    class SysNcursesConan(ConanFile):
+        name = "ncurses"
+        version = "system"
+        description = "A textual user interfaces that work across a wide variety of terminals"
+        topics = ("curses", "terminal", "toolkit")
+        homepage = "https://invisible-mirror.net/archives/ncurses/"
+        license = "MIT"
+        package_type = "shared-library"
+        settings = "os", "arch", "compiler", "build_type"
+
+        def package_id(self):
+            self.info.clear()
+
+        def validate(self):
+            if self.settings.os != "Linux":
+                raise ConanInvalidConfiguration(f"{self.ref} wraps a system package only supported by Linux.")
```

Comment:
looks like ncurses is also available for macOS with brew: https://formulae.brew.sh/formula/ncurses

Reply:
Updated on the commit [0be72f1](https://github.com/conan-io/docs/pull/3978/commits/0be72f1be4fa9dbf7fb982b9a89d35f4c3bcc7f0). Also added support for FreeBSD:

https://github.com/conan-io/examples2/pull/177#issuecomment-2630822064